src/pages/contrato.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import Link from 'next/link';
+import Head from '../components/head';
+import linebreaksbr from '../lib/linebreaksbr';
+import { getContract } from '../lib/api';
+
+class Contratos extends React.Component {
+  static async getInitialProps({ query }) {
+    let slug = query.slug;
```

Comment:
nit: use const

Reply:
prefer let in this case https://www.npmjs.com/package/eslint-plugin-prefer-let

Reply:
👍